Packs/HYASInsight/ReleaseNotes/1_0_3.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### HYAS Insight
```

Comment:
I recommend that you add a section that describes the changes you made related to WHOIS commands (URL and endpoint updates).